Examples/test-suite/autodoc.i

```diff
@@ -148,3 +148,11 @@ bool is_python_builtin() { return true; }
 bool is_python_builtin() { return false; }
 #endif
 %}
+
+// Autodoc Python keywords
+%warnfilter(SWIGWARN_PARSE_KEYWORD) process;
+%feature(autodoc,0) process;
+%feature("compactdefaultargs") process;
+%inline %{
+int process(int from) { return from; }
+%}
```

Comment:
Could you please add more tests combining "kwargs" and "compactdefaultargs" using keywords in the argument list? And also use optional parameters in the C++ function with keywords.